### PR TITLE
Adding VHAL properties in sepolicy

### PR DESCRIPTION
**Pull request by gkdeepa**
Tracked-On:
Signed-off-by: gkdeepa g.k.deepa@intel.com

### CODE REVIEW

**Review by xihua-chen — APPROVED** [review]

File: camera-ext/ext-camera-only/property.te [hunk 1]
```diff
@@ -0,0 +1,2 @@
+vendor_internal_prop(vendor_camera_default_prop)
+set_prop(vendor_init, vendor_camera_default_prop)
```
Comment (ceiba1985) [inline camera-ext/ext-camera-only/property.te]: move this to vendor_init.te

File: camera-ext/ext-camera-only/hal_camera_default.te [hunk 2]
```diff
@@ -6,3 +6,7 @@ allow hal_camera_default gpu_device:dir search;
 allow hal_camera_default hal_graphics_allocator_default_tmpfs:file { map read write };
 allow hal_camera_default hal_graphics_mapper_hwservice:hwservice_manager find;
 allow hal_camera_default hal_graphics_composer_default:fd use;
+allow hal_camera_default self:{ socket vsock_socket } { create read write listen accept bind };
+get_prop(hal_camera_default, vendor_camera_default_prop)
+set_prop(vendor_init, vendor_camera_default_prop)
```
Comment (ceiba1985) [inline camera-ext/ext-camera-only/hal_camera_default.te]: move this to vendor_init.te

File: aafd/logwrapper.te [hunk 3]
```diff
@@ -13,6 +13,7 @@ set_prop(logwrapper, vendor_mount_ep0_prop)
 set_prop(logwrapper, vendor_graphics_hwcomposer_prop)
 set_prop(logwrapper, vendor_video_codec_prop)
 set_prop(logwrapper, vendor_audio_default_prop)
+set_prop(logwrapper, vendor_camera_default_prop)
```
Comment (ceiba1985) [inline aafd/logwrapper.te]: move this to the dir where vendor_camera_default_prop defined.

**Review by ceiba1985 — APPROVED** [review]

File: camera-ext/ext-camera-only/hal_camera_default.te [hunk 4]
```diff
@@ -6,3 +6,7 @@ allow hal_camera_default gpu_device:dir search;
 allow hal_camera_default hal_graphics_allocator_default_tmpfs:file { map read write };
 allow hal_camera_default hal_graphics_mapper_hwservice:hwservice_manager find;
 allow hal_camera_default hal_graphics_composer_default:fd use;
+allow hal_camera_default self:{ socket vsock_socket } { create read write listen accept bind };
+get_prop(hal_camera_default, vendor_camera_default_prop)
+set_prop(hal_camera_default, vendor_camera_default_prop)
+set_prop(vendor_init, vendor_camera_default_prop)
```
Comment (ceiba1985) [inline camera-ext/ext-camera-only/hal_camera_default.te:12]: remove line 10 and line 12

**Review by ceiba1985 — APPROVED** [review]

**Pull request merged**